core/amber/src/main/python/core/architecture/sendsemantics/hash_based_shuffle_partitioner.py

```diff
@@ -42,6 +42,15 @@ def add_tuple_to_batch(
             yield receiver, batch
             self.receivers[hash_code] = (receiver, list())
 
+    @overrides
+    def add_state_to_batch(self, state: State):
```

Comment:
please annotate the return type. it is unclear.

Reply:
function removed.